ros_buildfarm/templates/snippet/builder_parameterized-trigger.xml.em

```diff
@@ -10,7 +10,7 @@
 @[if parameter_files]@
             <hudson.plugins.parameterizedtrigger.FileBuildParameters>
               <propertiesFile>@(','.join(ESCAPE(parameter_files)))</propertiesFile>
-              <failTriggerOnMissing>false</failTriggerOnMissing>
+              <failTriggerOnMissing>@(vars().get('missing_parameter_files_skip', False) ? 'true' : 'false')</failTriggerOnMissing>
```

Comment:
This parameter essentially causes the trigger operation to be skipped if the file isn't present. This is the case if the package is architecture-independent, or doesn't generate a symbol package for some other reason.